consensus/src/chained_bft/consensus_types/vote_msg.rs

```diff
@@ -60,9 +65,24 @@ impl VoteMsg {
             author,
             ledger_info: ledger_info_placeholder,
             signature: li_sig.into(),
+            round_signature: None,
         }
     }
 
+    /// Generates a round signature, which can then be used for aggregating a timeout certificate.
+    /// Typically called for generating vote messages that are sent upon timeouts.
+    pub fn add_round_signature(&mut self, validator_signer: &ValidatorSigner) {
```

Comment:
This is only for testing I think, since it's only used there, it might be nice to remove this method and just instantiate a new one to preserve immutability.

Reply:
Oh, we're not doing it in this PR, but in the followup we're going to add the round signature to the `last_sent_vote` message that is kept in the EventProcessor upon timeout.